src/lib/maplayers.js

```diff
@@ -150,17 +159,27 @@ class MapLayers {
 
   getTigerHistoricalRangeLayer(species) {
     const speciesSlug = species.name.replace(' ', '_')
+
     const url = `${TILE_API_ROOT}/tiles/species/${speciesSlug}/{z}/{x}/{y}/`
 
     return new SecureTileLayer(url, this.defaultTileLayerConfig)
   }
 
   getStructuralHabitat(species, date, setClosestDate) {
+    const extraOptions = this.defaultTileLayerConfig
```

Comment:
since you are doing the same thing twice, it's worth having another method to DRY this up